cylc/flow/scheduler.py

```diff
@@ -1232,7 +1232,8 @@ def process_task_pool(self):
             itask.waiting_on_job_prep
         ]
 
-        if self.stop_mode is None and self.auto_restart_time is None:
+        if (not self.is_paused and
+                self.stop_mode is None and self.auto_restart_time is None):
```

Comment:
I'm not sure if the `if not self.is_paused` should also cover the block above @hjoliver 
```python
# Forget tasks that are no longer preparing for job submission.
self.pre_submit_tasks = [
    itask for itask in self.pre_submit_tasks if
    itask.waiting_on_job_prep
]
```

Reply:
Job prep completion happens asynchronously, via subprocesses, so it can complete even if the flow is paused.  On the other hand, nothing will submit anyway if paused ... so I don't think it matters much either way.  But, maybe cover that block too, just for a tiny efficiency advantage.